tasks/firewalld.yml

```diff
@@ -6,12 +6,12 @@
     state: started
     enabled: yes
 
-- name: Configure firewall
-  firewalld:
-    permanent: true
-    immediate: true
-    service: "{{ item }}"
-    state: enabled
-  with_items:
-    - http
-    - https
+# - name: Configure firewall
```

Comment:
it is required, I just commented it because of travis docker installation, if we want to merge that we should necessary uncomment this task to allow HTTP and HTTPS traffic

Reply:
Please uncomment it. :smile: